src/bicep/examples/containerRegistry/README.md

```diff
@@ -0,0 +1,63 @@
+# Azure Container Registry Example
+
+This example deploys a premium Azure Container Registry suitable for hosting docker containers.  The registry will be deployed to the MLZ shared services resource group using default naming unless alternative values are provided at run time.
+
+Read on to understand what this example does, and when you're ready, collect all of the pre-requisites, then deploy the example.
+
+## What this example does
+
+### Deploys an Azure Container Registry
+
+The docs on Azure Container Registry: <https://docs.microsoft.com/en-us/azure/container-registry/>.  This sample shows how to deploy using Bicep and utilizes the shared file variable pattern to support the deployment.  By default, this template will deploy resources into standard default MLZ subscriptions and resource groups.  
+
+The subscription and resource group can be changed by providing the resource group name (Param: targetResourceGroup) and ensuring that the Azure context is set the proper subscription.  
+
+## Pre-requisites
+
+1. A Mission LZ deployment (a deployment of mlz.bicep)
+2. The outputs from a deployment of mlz.bicep (./src/bicep/examples/deploymentVariables.json).  
+
+See below for information on how to create the appropriate deployment variables file for use with this template.
+
+### Template Parameters
+
+Template Parameters Name | Description
+-----------------------| -----------
+contRegistryName | The name of the Container Registry.  If not specified, the name will default to the MLZ default naming pattern.  
+targetResourceGroup | The name of the resource group where the Container Registry will be deployed.   If not specified, the resource group name will default to the shared services MLZ resource group name and subscription.
+
+### Generate MLZ VAriable File (deploymentVariables.json)
+
+For instructions on generating 'deploymentVariables.json' using both Azure PowerShell and Azure CLI, please see the [README at the root of the examples folder](..\README.md).
```

Comment:
The link is broken--change to a forward slash. 

Also, the README at that location does not contain instructions on generating the `deploymentVariables.json` file.

```suggestion
For instructions on generating 'deploymentVariables.json' using both Azure PowerShell and Azure CLI, please see the [README at the root of the examples folder](../README.md).
```

Reply:
The README.md in the root of the examples folder was updated as a part of a different PR.  Should be good to go now.